### PR TITLE
MCP-1932: examOrderingStatus controller shell

### PR DESCRIPTION
**Pull request by dimitri-amida**
# Feature Name 

### What was the problem?

see (https://amida.atlassian.net/browse/MCP-1932)

### Jira Tickets

- [MCP-1932](https://amida.atlassian.net/browse/MCP-1932)

## How to test this PR

- New endpoint

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
- [ ] If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) and
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
- [ ] If changing software behavior, post a link to the PR in Slack channel #benefits-virtual-regional-office

### CODE REVIEW

File: api/src/main/java/gov/va/vro/api/resources/MasResource.java [hunk 1]
```diff
@@ -61,4 +62,36 @@ ResponseEntity<MasClaimResponse> notifyAutomatedClaimDetails(
...
+        @ApiResponse(
+            responseCode = "401",
+            description = "Unauthorized",
+            content = @Content(schema = @Schema(hidden = true))),
+        @ApiResponse(
+            responseCode = "500",
+            description = "Data Access Server Error",
+            content = @Content(schema = @Schema(hidden = true))),
+        @ApiResponse(
+            responseCode = "404",
```
Comment (au2) [inline api/src/main/java/gov/va/vro/api/resources/MasResource.java:83]: Are we going to have 404 response really?

File: api/src/main/java/gov/va/vro/api/resources/MasResource.java [hunk 2]
```diff
@@ -61,4 +62,36 @@ ResponseEntity<MasClaimResponse> notifyAutomatedClaimDetails(
...
+            description = "No evidence found",
+            content = @Content(mediaType = "application/json"))
+      })
+  @Timed(value = "exam-ordering-status")
+  @Tag(name = "MAS Exam Ordering Status")
```
Comment (au2) [inline api/src/main/java/gov/va/vro/api/resources/MasResource.java:88]: Can we just use the same tag for MAS end point for now? Maybe "MAS Integration"? You need to do the same change in OpenAPIConfiguration.java

File: domain/src/main/java/gov/va/vro/model/mas/MasOrderingStatusPayload.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  @Schema(description = "Collection ID", example = "999")
+  private String collectionId;
+
+  @Schema(description = "Collection Status", example = "DRAFT")
+  private String collectionStatus;
+
+  @Schema(
+      description = "Exam order timestamp",
+      example =
+          "225-63-70T34:63:40.566614908642876440526533774826377332727118627060130905913+84:54")
```
Comment (au2) [inline domain/src/main/java/gov/va/vro/model/mas/MasOrderingStatusPayload.java:28]: Why don't we have an actual ISO date here as an example?

**Review by au2 — CHANGES_REQUESTED** [review]

File: api/src/main/java/gov/va/vro/api/resources/MasResource.java (hunk 1 above)
Comment (dimitri-amida) [reply api/src/main/java/gov/va/vro/api/resources/MasResource.java:83]: I don't know. @rajesh-Krishnasamy ?

File: domain/src/main/java/gov/va/vro/model/mas/MasOrderingStatusPayload.java (hunk 3 above)
Comment (dimitri-amida) [reply domain/src/main/java/gov/va/vro/model/mas/MasOrderingStatusPayload.java:28]: It does look weird but I copied this exactly from the open API docs. @rajesh-Krishnasamy ?

**Review by rajesh-Krishnasamy — APPROVED** [review]

**Review by wlin2019 — APPROVED** [review]

File: domain/src/main/java/gov/va/vro/model/mas/MasOrderingStatusPayload.java (hunk 3 above)
Comment (dimitri-amida) [reply domain/src/main/java/gov/va/vro/model/mas/MasOrderingStatusPayload.java]: I fixed it

**Comment by dimitri-amida** [comment]
@rajesh-Krishnasamy I updated to the latest API spec. Is there anything more to do on this ticket for Exam Order Status, or is it just the controller definition. @au2 I believe I addressed all your comments.

**Comment by au2** [comment]
@dimitri-amida can you resolve conflicts?

**Comment by dimitri-amida** [comment]
@au2 conflicts resolved

**Review by au2 — APPROVED** [review]

**Pull request merged**